src/Infrastructure/Persistence/Interceptors/DispatchDomainEventsInterceptor.cs

```diff
@@ -7,18 +7,32 @@ namespace SSW.CleanArchitecture.Infrastructure.Persistence.Interceptors;
 
 public class DispatchDomainEventsInterceptor(IMediator mediator) : SaveChangesInterceptor
 {
-    public override InterceptionResult<int> SavingChanges(DbContextEventData eventData, InterceptionResult<int> result)
+    // NOTE: There are two options for dispatching domain events:
```

Comment:
@william-liebenberg @matt-goldman-ssw - you guys were looking into this some time back and may be interested in this decision.

Reply:
Still a pending effort on this one :)